src/routes/auth/activate.ts

```diff
@@ -49,7 +49,10 @@ async function activateUser({ query }: Request, res: Response): Promise<unknown>
     return res.redirect(302, APPLICATION.REDIRECT_URL_SUCCESS)
   }
 
-  res.status(200).send('Your account has been activated. You can close this window and login')
+  if(APPLICATION.REDIRECT_URL_SUCCESS) {
+    res.redirect(APPLICATION.REDIRECT_URL_SUCCESS.replace('JWT_TOKEN', ticket))
```

Comment:
What's the use case here? I feel like `JWT_TOKEN` and `ticket` should not be mixed like this?

Reply:
Let's fix in a separate issue